solutions/ac-MultipleOf2019.mdx

```diff
@@ -47,5 +47,41 @@ int main()
 ```
 
 </CPPSection>
+	
+<JavaSection>
+	
+```java
+import java.io.*;
+import java.util.*;
+
+public class MultipleOf2019 {
+	public static void main (String[] args) {
+		Kattio io = new Kattio();
+		String numberString = io.next();
```

Comment:
just `num` would be ok

Reply:
num is used as the name of another variable

Reply:
numberString is more clear

Reply:
yeah sure